### PR TITLE
Added Json Web Token (JWT) Authenticator

### PR DESCRIPTION
**Pull request by rkravchik**
Added Json Web Token (JWT) Authenticator
https://tools.ietf.org/html/draft-jones-json-web-token

### CODE REVIEW

File: RestSharp/Authenticators/JwtAuthenticator.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#region License
```
Comment (hallem) [inline RestSharp/Authenticators/JwtAuthenticator.cs]: Please remove your personal license

**Comment by hallem** [comment]
Thanks for the PR.  Can you write some tests to show that your code works please?  Once the tests are in I'll merge the PR.

**Comment by hallem** [comment]
@rkravchik you need to remove your personal license from the commit and add unit tests before i can merge this in.

**Comment by rkravchik** [comment]
ОК.
I've tried to connect XUnit to my SharpDevelop last weekends.
It needs me just a few days to finish with a tests.

**Comment by rkravchik** [comment]
I'm not sure about adding main files to *.csproj. Is it ok?

**Pull request merged**